src/twisted/python/text.py

```diff
@@ -7,8 +7,11 @@
 Miscellany of text-munging functions.
 """
 
+import typing
```

Comment:
import `IO` in the list below instead.

Reply:
(Or `TextIO`)

Reply:
I'd rather have the `IO` type from `typing` be explicitly qualified, because it's easy to confuse it with the more concrete imports from the `io` module. It should be clear at the point of use that it refers to an arbitrary "file-like object" rather than something that descends from a particular class.